File: examples/js/db/update/updateARecord.js
const { DbService } = require("m3o/db");

// Update a record in the database. Include an "id" in the record to update.
async function updateArecord() {
  let dbService = new DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.update({
    record: {
      age: 43,
      id: "1",
    },
    table: "users",
  });
  console.log(rsp);
}

updateArecord();

File: examples/go/db/update/updateARecord.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/db"
)

// Update a record in the database. Include an "id" in the record to update.
func UpdateArecord() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.Update(&db.UpdateRequest{
		Record: map[string]interface{}{
			"id":  "1",
			"age": 43,
		},
		Table: "users",
	})
	fmt.Println(rsp, err)
}
